### PR TITLE
bug(refs: t34576): add missing icon and badge

### PR DESCRIPTION
**Pull request by muellerdemos**
https://yaits.demos-deutschland.de/T34576

The icon is used as a trigger for the tooltip
The badge transports the information that this feature is currently in beta state

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by gruenbergerdemos — CHANGES_REQUESTED** [review]
The tooltip appears behind the modal

File: client/js/components/procedure/StatementSegmentsList/StatementSegment.vue [hunk 1]
```diff
@@ -118,10 +118,22 @@
               ref="recommendationModal"
               class="recommendation-modal"
               content-classes="u-2-of-3">
-              <div class="flex width-100p">
+              <div
+                class="flex width-100p">
```
Comment (salisdemos) [inline client/js/components/procedure/StatementSegmentsList/StatementSegment.vue:122]:
```suggestion
              <div class="flex width-100p">
```

**Comment by muellerdemos** [comment]
@gruenbergerdemos the tooltip appearing behind layouts appears to be a known problem which hopefully resolves with T34571

File: client/js/components/procedure/StatementSegmentsList/StatementSegment.vue (hunk 1 above)
Comment (muellerdemos) [reply client/js/components/procedure/StatementSegmentsList/StatementSegment.vue]: done here: https://github.com/demos-europe/demosplan-core/pull/1967/commits/f3c23e0afd9518c767502e6dd25fb6d888efdc89

**Review by gruenbergerdemos — APPROVED** [review]
lgtm, thanks

**Pull request merged**